prompting/apparmor/permission.go

```diff
@@ -81,7 +81,7 @@ const (
 	MayChangeProfilePermission FilePermission = 1 << 30
 )
 
-const filePermissionMask = (MayExecutePermission | MayWritePermission | MayReadPermission |
+const AllFilePermissionMask = (MayExecutePermission | MayWritePermission | MayReadPermission |
```

Comment:
`AllFilePermissionMask` is newly exported, but this seems reasonable, so that the `common` package can reliably convert from apparmor `FilePermission` to common `PermissionType` with the same error handling.

Reply:
I unexported this and instead use the `IsValid()` method where relevant. This PR was merged before this change, but the commit is 8708326912136f86630c2b1b8aa60b7021e78069